### PR TITLE
change border color

### PR DESCRIPTION
**Pull request by joel-molero**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <style>
         body{
             background: orange;
-            border: 10px solid skyblue;
```
Comment (Sergioluz90) [inline index.html:11]: Greenyellow sucks

Comment (joel-molero) [reply index.html:11]: Be careful, I have access to the same coffee machine as you.

File: index.html [hunk 2]
```diff
@@ -8,7 +8,7 @@
     <style>
```
Comment (Sergioluz90) [inline index.html:8]: hey, move styles to file!

Comment (joel-molero) [reply index.html:8]: You do it. I'm too lazy.

Comment (Sergioluz90) [reply index.html:8]: No problema, I proceed to move styles to file, after merge. You Will pay me at lunch.

Comment (joel-molero) [reply index.html:8]: Get a job mate.

**Pull request merged**